flow360/component/flow360_params/solvers.py

```diff
@@ -186,17 +186,25 @@ def _get_optional_objects(cls) -> List[str]:
         return ["linearSolver"]
 
 
-class TurbulenceModelConstants(Flow360BaseModel):
-    """:class:`TurbulenceModelConstants` class"""
+class TurbulenceModelConstantsSA(Flow360BaseModel):
+    """:class:`TurbulenceModelConstantsSA` class"""
 
     C_DES: Optional[float]
     C_d: Optional[float]
+
+
+class TurbulenceModelConstantsSST(Flow360BaseModel):
+    """:class:`TurbulenceModelConstantsSST` class"""
+
     C_DES1: Optional[float]
     C_DES2: Optional[float]
     C_d1: Optional[float]
     C_d2: Optional[float]
 
 
+TurbulenceModelConstants = Union[TurbulenceModelConstantsSA, TurbulenceModelConstantsSST]
```

Comment:
`TurbulenceModelConstants` seems not be used, right?

Reply:
it is used in the base class of turbulence model solver, but it is useless, so removed.

Reply:
restored due to model_constants is required for legacy